packages/dapp-svelte-wallet/react-ui/package.json

```diff
@@ -45,8 +46,14 @@
     "lint-check": "eslint '**/*.{js,jsx}'",
     "lint-fix": "eslint --fix '**/*.{js,jsx}'",
     "test": "react-scripts test",
+    "test:debug": "react-scripts --inspect-brk test --runInBand --no-cache",
     "eject": "react-scripts eject"
   },
+  "jest": {
+    "transformIgnorePatterns": [
+      "/node_modules/(?!@lit-labs)/"
```

Comment:
`react-scripts test` uses `jest` which does not support es6 modules. It uses babel under the hood to transpile es6 to commonjs, but by default this ignores node_modules, so we have to enable transpilation for certain modules else it complains with "unexpected token export".